lib/sdk/di/injector.dart

```diff
@@ -186,10 +188,15 @@ abstract class EncryptionModule {
 }
 
 @module
-abstract class ZipDecoderModule {
+abstract class FilesManagerModule {
   @Named('zipDecoder')
   @factoryMethod
   ZipDecoder zipDecoder() {
     return ZipDecoder();
   }
+
+  @lazySingleton
+  @preResolve
+  Future<Directory> get applicationDocumentsDirectory async =>
+      await getApplicationDocumentsDirectory();
```

Comment:
Do you need to preresolve it?
```suggestion
  Future<Directory> get applicationDocumentsDirectory => getApplicationDocumentsDirectory();
```